addons/website/static/src/js/tours/tour_utils.js

```diff
@@ -79,8 +79,12 @@ function changeImage(snippet, position = "bottom") {
 function changeOption(optionName, weName = '', optionTooltipLabel = '', position = "bottom", allowPalette = false) {
     const noPalette = allowPalette ? '' : '.o_we_customize_panel:not(:has(.o_we_so_color_palette.o_we_widget_opened))';
     const option_block = `${noPalette} we-customizeblock-option[class='snippet-option-${optionName}']`;
+    // The data-original-title and the title are necessary because sometimes the
+    // tour does not click fast enough on the element after hover it. The
+    // element has then the time to initialize its tooltips, this one will put
```

Comment:
Thanks for the comment! I am really not convinced about the explanation, after re-thinking about it.
A tour is not doing `clicks` in a human way so there is no hover before click.
You had test failing without it, right?

```suggestion
    // element has then the time to initialize its tooltips, this one will move
```